### DISCUSSION TITLE
how can redirect when next config was moved to middleware

### DISCUSSION THREAD

**Discussion by swajp** [amannn/next-intl#1395] [Q&A]
i was redirecting a user to other page in next.config.mjs but that is not supported anymore how can i do that in the newer version?

```
const nextConfig = {
    i18n: {
        locales: ["default", "en", "cs", "sk", "hu"],
        defaultLocale: "default",
        localeDetection: false
    },
      trailingSlash: true,
    async redirects() {
        return [
            {
                source: "/case-studies/edu",
                destination: "/case-studies/my-edu-study",
                permanent: true
            },
            {
                source: "/case-studies/pb",
                destination: "/case-studies/my-prague-study",
                permanent: true
            }
        ]
    }
}
```

**Comment by swajp** [comment]
@amannn please any suggestion?

**Reply by amannn** [reply]
You can absolutely still redirect via `redirects` in `next.config.js` (see the [docs](https://nextjs.org/docs/app/api-reference/next-config-js/redirects)). You also have [other options](https://nextjs.org/docs/app/building-your-application/routing/redirecting) though if you prefer one of those.

 Maybe what you're looking for is considering a locale prefix for a redirect?
